docs/security/05-05-production-profile.md

```diff
@@ -1,65 +1,53 @@
 ---
-title: OAuth2 server production profile
+title: OAuth2 server profiles
 type: Configuration
 ---
 
-By default, every Kyma deployment is installed with the OAuth2 server using what is considered a development profile. In the case of the ORY Hydra OAuth2 server, this means that:
```

Comment:
An introduction is missing here. Please write a brief introductory sentence describing the profiles we have and when we use them.